tests/storage-vm

```diff
@@ -484,6 +484,29 @@ for poolDriver in $poolDriverList; do
 
         lxc delete -f v1
 
+        echo "==> Checking start / stop operations with root size.state being unset"
```

Comment:
@gabrielmougard this should check the same API extension as https://github.com/canonical/lxd-ci/pull/75 so it skips on 5.0/edge cc @simondeziel

Reply:
I'll try to do a better job at catching those API checks, thanks for being so vigilant Tom.

Reply:
fixed